### ISSUE TITLE
Global Styles

### ISSUE THREAD

**Issue by penx** [govuk-react/govuk-react#424]
This issue is to discuss how we define global style, what would be contained in global styles (e.g. fonts, reset), to agree on a standard and to implement any code or documentation updates needed for that standard.

We could distribute some common settings or define how a parent application would implement this manually in the README.

Things to consider:

- gov.uk domains vs. non gov.uk domains
- CSSinJS (injectGlobal) or static CSS file
- server rendered, client rendered or universal
- Use a published CSS reset or use a basic/custom one
- perhaps some sites want to fall back to 100% system fonts e.g. for 100% unicode support
- non govuk domains - official fallback vs. custom fonts
- how to preview in storybook

**Comment by penx** [comment]
> There are 2 ways to do this:
> Declaring your global styles (reset, normalize, etc) in a separate file.
> This escape hatch: (injectGlobal)
> 
> I suggest using a file. injectGlobal works best when it is called once on the server (at the right moment) and is not called on the client.


https://medium.com/@wonderboymusic/a-field-guide-to-css-in-js-f7cbd9ed79a7

**Comment by penx** [comment]
@Loque- had done the following in a CSS file manually in a prototype:

```css
@import url('https://rsms.me/inter/inter-ui.css');

*,
*::before,
*::after {
  -webkit-box-sizing: border-box;
  box-sizing: border-box;
}

html {
  -webkit-text-size-adjust: 100%;
}

img,
svg {
  vertical-align: top;
}

html,
body {
  padding: 0;
  margin: 0;
  background: #f3f3f3;
  font-family: 'Inter UI', Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
```

**Comment by penx** [comment]
planning to use what I have in the example app

**Comment by penx** [comment]
Fixed in #834
